common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -85,7 +93,7 @@ public Object getValue() {
     public boolean setValue(Object value) {
         try {
             FieldUtils.writeField(field, parent, value, true);
-            parent.updateConfigOption(this);
+            ((Configurable) parent).updateConfigOption(this);
```

Comment:
Actually, I think it *would* make sense to declare parent as `Configurable`.

Reply:
... and maybe Configurable should implement/extend Translatable... I realize it's a bit of mixing of concerns, but there is no good way in Java either to require that the parent sent to the constructor is both Configurable and Translatable.

Reply:
Make parent Configurable and throw RuntimeException in ctor if it is not Translatable? I don't want to mix the two into a singular thing?

@magicus, do you accept this?